scripts/deploy/update-npm-tag.js

```diff
@@ -1,7 +1,7 @@
 const {promisify} = require('util');
 const exec = promisify(require('child_process').exec);
 
-const packageDirs = ['atomic', 'bueno', 'headless'];
+const packageDirs = ['atomic', 'auth', 'bueno', 'headless'];
```

Comment:
An alternative here could be to find all dirs inside `packages/`, excluding `quantic` and `samples`.
On the other hand, the current white list might be clearer to visualize what packages are being updated.

Open to either,

Reply:
I personally prefer a white list like this.